package.json

```diff
@@ -23,27 +23,27 @@
         "lint": "jshint *.js packages",
         "test": "mocha -R spec --require ts-node/register --grep E2E --invert",
         "test:unit": "nyc --no-clean --silent _mocha -- -R spec --require ts-node/register --grep E2E --invert --exit",
-        "test:cov": "npm run cov:clean; npm run test:unit; npm run test:e2e:clients; npm run cov:merge_reports",
+        "test:cov": "npm run cov:clean && npm run test:unit && npm run test:e2e:clients && npm run cov:merge_reports",
         "dtslint": "lerna run dtslint",
         "depcheck": "lerna exec dependency-check -- --missing --verbose .",
         "bundlesize": "bundlesize || true",
         "geth": "geth-dev-assistant --accounts 5 --tag stable --gasLimit 7000000",
-        "test:e2e:ganache": "./scripts/e2e.ganache.sh",
-        "test:e2e:geth:auto": "./scripts/e2e.geth.automine.sh",
-        "test:e2e:geth:insta": "./scripts/e2e.geth.instamine.sh",
-        "test:e2e:clients": "npm run test:e2e:ganache; npm run test:e2e:geth:insta; npm run test:e2e:geth:auto",
-        "test:e2e:chrome": "./scripts/e2e.chrome.sh",
-        "test:e2e:firefox": "./scripts/e2e.firefox.sh",
-        "test:e2e:min": "./scripts/e2e.min.sh",
-        "test:e2e:cdn": "./scripts/e2e.cdn.sh",
-        "test:e2e:browsers": "npm run build; npm run test:e2e:chrome; npm run test:e2e:firefox",
-        "test:e2e:publish": "./scripts/e2e.npm.publish.sh",
-        "test:e2e:truffle": "./scripts/e2e.truffle.sh",
-        "test:e2e:mosaic": "./scripts/e2e.mosaic.sh",
-        "test:e2e:ganache:core": "./scripts/e2e.ganache.core.sh",
-        "test:e2e:gnosis:dex": "./scripts/e2e.gnosis.dex.sh",
-        "ci": "./scripts/ci.sh",
-        "cov:clean": "rm -rf .nyc_output; rm -rf coverage",
+        "test:e2e:ganache": "bash ./scripts/e2e.ganache.sh",
+        "test:e2e:geth:auto": "bash ./scripts/e2e.geth.automine.sh",
+        "test:e2e:geth:insta": "bash ./scripts/e2e.geth.instamine.sh",
+        "test:e2e:clients": "npm run test:e2e:ganache && npm run test:e2e:geth:insta && npm run test:e2e:geth:auto",
+        "test:e2e:chrome": "bash ./scripts/e2e.chrome.sh",
+        "test:e2e:firefox": "bash ./scripts/e2e.firefox.sh",
+        "test:e2e:min": "bash ./scripts/e2e.min.sh",
+        "test:e2e:cdn": "bash ./scripts/e2e.cdn.sh",
+        "test:e2e:browsers": "npm run build && npm run test:e2e:chrome && npm run test:e2e:firefox",
+        "test:e2e:publish": "bash ./scripts/e2e.npm.publish.sh",
+        "test:e2e:truffle": "bash ./scripts/e2e.truffle.sh",
+        "test:e2e:mosaic": "bash ./scripts/e2e.mosaic.sh",
+        "test:e2e:ganache:core": "bash ./scripts/e2e.ganache.core.sh",
+        "test:e2e:gnosis:dex": "bash ./scripts/e2e.gnosis.dex.sh",
+        "ci": "bash ./scripts/ci.sh",
+        "cov:clean": "rimraf .nyc_output && rimraf coverage",
```

Comment:
```suggestion
        "cov:clean": "rm -rf .nyc_output; rm -rf coverage",
```

Reply:
The suggested change actively reduces cross-platform compatibility.  Can you explain why it's important to avoid any changes to package-lock?

Reply:
@wbt We include changes in package-lock files via dedicated and internally initialized PRs only.

Reply:
This is now formally an internally initialized PR.  Going out of your way to reduce cross-platform compatibility and break the command on some platforms just doesn't seem like the right direction.

Reply:
`rimraf` works exactly the same as `rm-rf` but with cross-platform support. Why do we need to change `rimraf` to `rm -rf`? What is it solve?

Reply:
> @wbt We include changes in package-lock files via dedicated and internally initialized PRs only.

Internal and also **dedicated** PRs for package lock changes. Thanks

Reply:
I should state agreement that I don't think all of the changes to package-lock files, especially those that alter the root package version number, should be included in this PR, but I also disagree with the very broadly stated rule that package-lock **cannot change at all** in any PR because **some** package-lock changes accompany any updates to the sets of active dependencies.

Reply:
However, the version of this PR which does not change all the package-lock files this one does was rejected at #4848.  In theory, I believe that one could be reopened and merged, but @spacesailor24 seems to prefer the version with all the additional package-lock changes while @jdevcs strongly disagrees, and meanwhile the build command nominally required to be run for every PR remains broken.

Reply:
@wbt , we discussed this internally that @spacesailor24 will re generate package lock files in this PR, but there are errors when he tried. https://github.com/ChainSafe/web3.js/pull/4861#issuecomment-1078599638 

Our team is occupied in [4.x](https://github.com/ChainSafe/web3.js/tree/4.x) so could you investigate and share it.

Reply:
@jdevcs can you just reopen #4848?

Reply:
Its already merged so couldn't reopen.

Reply:
Replaced with #4939.